src/components/shared/user-courses/share/progress-status.scss

```diff
@@ -0,0 +1,17 @@
+$progress-bar-color: #000;
```

Comment:
Why do you define a variable and don't use it? also, you can use _variables.scss for defining your constant variables

Reply:
Actually it's not an unused variable. This variable directly manipulate the Progressbar component of ParagonDesignSystem. As per Paragon documentation this is the best way to customize (change color and background and fonts). And it is recommended that to customize every component styles just create its own SASS classes in the same directory. When you create your own variable inside _variables.scss it would be global to the entire project.